utils/nctl/sh/scenarios/configs/itst13.config.toml

```diff
@@ -6,6 +6,8 @@
 # If set, use this hash as a trust anchor when joining an existing network.
 #trusted_hash = 'HEX-FORMATTED BLOCK HASH'
 
+archival_sync = false
```

Comment:
```suggestion
archival_sync = true
```

Reply:
With the current implementation, this will cause nodes to take too long to join.

After implementing your proposal I have outlined in https://github.com/casper-network/casper-node/issues/2083 we can set these flags to `true`.